pkg/agent/openflow/client.go

```diff
@@ -240,6 +240,31 @@ type Client interface {
 	IsIPv4Enabled() bool
 	// Returns if IPv6 is supported on this Node or not.
 	IsIPv6Enabled() bool
+	// SendTCPPacketOut sends TCP packet as a packet-out to OVS.
+	SendTCPPacketOut(
```

Comment:
I am sure we can improve the interface for these, but let's do it in a future dedicated PR, which will cover `SendTraceflowPacket` as well.

Reply:
Sure thing.

Reply:
Also feel we should fix this. Probably we should move the funcs about generating a packet to the pkg/ovs/openflow, and just keep a SendPacket func in Openflow.Client.
Works for me to refactor later.